geode-core/src/main/java/org/apache/geode/distributed/internal/InternalConfigurationPersistenceService.java

```diff
@@ -373,22 +384,28 @@ public boolean removeJars(String[] jarNames, String[] groups) {
           break;
         }
 
-        for (String jarRemoved : jarNames) {
-          File jar = getPathToJarOnThisLocator(group, jarRemoved).toFile();
+        logger.debug("Configuration before removing deployment: " + configuration);
```

Comment:
Lots of debug logging here. Do you really want to keep all that?

Reply:
Yes, we found that without debug logging, it was hard debugging it whilst running.